docs/source/help-support.rst

```diff
@@ -1,35 +1,49 @@
-=================
+================
 Help and support
-=================
+================
 
-* Get free community support with our `Google group
-  <https://groups.google.com/a/anaconda.com/forum/#!forum/anaconda>`_.
+Free community support
+======================
 
-* Paid `support <https://www.anaconda.com/support/>`_,
-  `training <https://www.anaconda.com/training/>`_ and
-  `consulting <https://www.anaconda.com/consulting/>`_
-  options are available. Our teaching philosophy is that the best
-  way to learn is with hands-on experience addressing real-world
-  problems. Courses are available to individuals online, at
-  numerous sites, or in-house at your place of business. We also
-  offer consulting services for the analysis, management and
-  visualization of scientific and business data or optimizing
-  your processing workflows on modern hardware and GPUs.
+Ask and answer questions, discuss ways to use conda,
+request new features, and submit any other comments you might have.
```

Comment:
```suggestion
Use the following tools to ask and answer questions, discuss ways to use conda,
request new features, and submit any other comments you might have.
```

Should this be a complete sentence? Feels a little fragment-y to me.

Reply:
This is a great suggestion. Thanks!